src/main/scala/com/microsoft/hyperspace/actions/RefreshActionBase.scala

```diff
@@ -76,4 +78,56 @@ private[actions] abstract class RefreshActionBase(
           s"Current index state is ${previousIndexLogEntry.state}")
     }
   }
+
```

Comment:
Note to reviewers: `def deletedFiles` and `def appendedFiles` were picked directly from `RefreshDeleteAction` and `RefreshAppendAction` respectively.

The only change is, `IndexLogEntry.deletedFiles` is added to the return value of `def deletedFiles`.

In the same way, `IndexLogEntry.appendedFiles` is added to the return value of `def appendedFiles`.

Reply:
thanks @apoorvedave1 , noted!